bidrag-transport/src/main/kotlin/no/nav/bidrag/transport/behandling/beregning/felles/BidragBeregningResponsDto.kt

```diff
@@ -1,25 +1,27 @@
 package no.nav.bidrag.transport.behandling.beregning.felles
 
 import no.nav.bidrag.domene.enums.beregning.Samværsklasse
+import no.nav.bidrag.domene.enums.sak.Sakskategori
 import no.nav.bidrag.domene.enums.vedtak.Stønadstype
 import no.nav.bidrag.domene.ident.Personident
+import no.nav.bidrag.domene.tid.ÅrMånedsperiode
 import java.math.BigDecimal
 import java.time.LocalDate
 
 data class BidragBeregningResponsDto(
     val beregningListe: List<BidragBeregning>,
 ) {
     data class BidragBeregning(
+        val periode: ÅrMånedsperiode? = null,
         val saksnummer: String,
+        val sakskategori: Sakskategori = Sakskategori.N,
```

Comment:
Bør sakskategori ligge i denne responsen eller bør det hentes av orkestratoren? Det er en informasjon som ligger i saken så BBM/Behandling må kalle sak for å hente informasjonen uansett